build-support/bin/ci.py

```diff
@@ -276,29 +276,8 @@ def run_clippy() -> None:
 def run_cargo_audit() -> None:
     with travis_section("CargoAudit", "Running Cargo audit on Rust code"):
         try:
-            subprocess.run(
-                [
-                    "./cargo",
-                    "ensure-installed",
```

Comment:
When running locally, Cargo said `ensure-installed` wasn't installed. The main thing we seem to lose is pinning to a particular version, but I'm not sure that was desirable behavior. While pinning may make CI more stable, I'd rather that we stay abreast of any new vulnerabilities.

Reply:
We don't lose that, `cargo install` supports `--version`. Additionally it appears the reason for `ensure-installed`'s existence is now obsolete - cargo install now skips an install for already-installed crates with a 0 exit code:
```
$ ./cargo install --version 0.11.2 cargo-audit
  Downloaded cargo-audit v0.11.2
  Downloaded 1 crate (155.8 KB) in 1.02s
    Updating crates.io index
  Installing cargo-audit v0.11.2
  Downloaded thiserror-impl v1.0.23
  ...
    Finished release [optimized] target(s) in 1m 49s
  Installing /home/jsirois/.cargo/bin/cargo-audit
   Installed package `cargo-audit v0.11.2` (executable `cargo-audit`)
$ ./cargo install --version 0.11.2 cargo-audit
     Ignored package `cargo-audit v0.11.2` is already installed, use --force to override
```
So the technology supports pinning.

As to your second justification, it appears to be invalid. Looking here: https://github.com/RustSec/cargo-audit#screenshot you can see cargo-audit pulls down a vulnerability database from https://github.com/RustSec/advisory-db which appears to be independent of cargo-audit. That would seem to imply you can stay up-2-date with vulnerabilities while using a pinned version of cargo-audit.

Reply:
Yea, continuing to pin (but maybe to a more modern version) would be great.